pyproject.toml

```diff
@@ -7,6 +7,8 @@ license = "MIT"
 
```

Comment:
You should add a [script](https://python-poetry.org/docs/pyproject/#scripts) here that will call your Typer app; this will define an executable that will be installed with your program. See Typer's [run documentation](https://python-poetry.org/docs/pyproject/#scripts) for additional details. This is critical to get your application published to PyPI, so if you have any questions ask @mariakimheinert or myself as soon as you can.